site/src/components/ErrorSummary/ErrorSummary.tsx

```diff
@@ -51,18 +47,9 @@ export const ErrorSummary: FC<ErrorSummaryProps> = ({
   return (
     <Stack className={styles.root}>
       <Stack direction="row" alignItems="center" className={styles.messageBox}>
-        <div>
+        <div className={styles.flex}>
```

Comment:
Do we want to use another `Stack` component here so we don't have to add a class?